portfolio/src/main/webapp/index.html

```diff
@@ -44,85 +53,100 @@ <h2>Background</h2>
         </p>
       </div>
 
-      <br>
-      <h2>Projects:</h2>
-      <div id="projects">
-        <a href="https://drive.google.com/file/d/1Jb52Jk0HLoNbdVDg3z5VrQMrCsxFrkv-/view?usp=sharing" target="_blank">
-          <div class="proj-container">
-            <img class="proj-img" src="images/snapchat.jpg">
-            <p class="overlay">Snapchat Political Ads Spending vs Number of Views - Data Science Project</p>
+      <div id="proj-anchor"></div>
+      <div id="proj-container">
+        <h2>Projects:</h2>
+        <div id="projects">
+          <a href="https://drive.google.com/file/d/1Jb52Jk0HLoNbdVDg3z5VrQMrCsxFrkv-/view?usp=sharing" target="_blank" onmouseover="revealOnMouseover(this);" onmouseout="hideOnMouseout(this);">
+            <div class="proj-container">
+              <img class="proj-img background" src="images/snapchat.jpg">
+              <p class="overlay visible">Snapchat Political Ads Spending vs Number of Views - Data Science Project</p>
+              <p class="overlay hidden">Given the dataset for Snapchat Poltical Ads, we had to clean the data, perform EDA, 
+                  assess missingness in the data, and finally perform a hypothesis test which determines whether there is a 
+                  difference in the amount of ads targeting Spanish Speakers between the most expensive ads and the rest 
+                  of the ads.
```

Comment:
This sentence also seems very long. Can it be split?

Reply:
Done!